src/tasks/MailClaimFlow.py

```
@@ -0,0 +1,327 @@
+
+    def open_mail_panel(self):
+        """打开mail panel。"""
+        self.log_info("正在打开邮件面板")
+        panel = self.resolve_mail_phone_menu(self.open_esc_panel_for_mail())
+        clicked_phone_mail = self.click_mail_button_from_phone_menu(panel)
+        if not clicked_phone_mail:
+            self.click_ui(*self.MAIL_BUTTON_POSITION, after_sleep=1, move=True, down_time=0.01)
+        result = self.wait_panel(Labels.mail_panel, time_out=self.MAIL_PANEL_WAIT_TIMEOUT)
+        if not result:
+            if clicked_phone_mail:
+                self.log_info("邮件图标点击后未检测到邮件面板，重新打开手机菜单重试一次")
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)